mappings/translation_test_utilities.go

```diff
@@ -187,12 +187,9 @@ func assertTFieldBase(t *testing.T, field reflect.StructField, base models.Trans
 	assertStringPointerNilOrNotEmpty(t, base.SubLabel, "SubLabel", field)
 	assertStringPointerNilOrNotEmpty(t, base.MultiSelectLabel, "MultiSelectLabel", field)
 
-	// assert.NotZero(t, base.IsArray) // not zero doesn't work for bool because false is zero
-
-	assert.NotZero(base.DataType)
+	assert.NotZero(base.DataType, "issue for field %s. Value: %s", field.Name)
```

Comment:
Should this be using `NotZerof` if you want to use formatted strings?

Reply:
yes! Good call, thanks, I'll update these method calls.